src/main/resources/avro/sequenceAnnotations.avdl

```diff
@@ -0,0 +1,227 @@
+@namespace("org.ga4gh.models")
+/**
+This protocol defines annotations on GA4GH genomic sequences It includes two
+types of annotations: continuous and discrete hierarchical.
+
+The discrete hierarchical annotations are derived from the Sequence Ontology
+(SO) and GFF3 work 
+
+   http://www.sequenceontology.org/gff3.shtml
+
+The goal is to be able to store annotations using the GFF3 and SO conceptual
+model, although there is not necessarly a one-to-one mapping in Avro records
+to GFF3 records.
+
+The minimum requirement is to be able to accurately represent the current
+state of the art annotation data and the full SO model.  Feature is the
+core generic record which corresponds to the a GFF3 record.
+*/
+protocol SequenceAnnotations {
+
+  import idl "common.avdl";
+  import idl "ontologies.avdl";
+
+  /**
+  Identifier from a public database
+  */
+  record ExternalIdentifier {
```

Comment:
This seems either like it would be extremely useful all over the place (`rs` numbers for variants!) or that it should somehow be encoded as a string typedef (ex: `Ensembl:ENST00000000000:v78`).